1-user.md

```diff
@@ -88,7 +88,7 @@ any yaml file at /etc/netplan: (change interface names, ip ranges accordingly)
            addresses: [192.168.122.10/24]
            routes:
              - to: default
-               from: 192.168.122.1
+               via: 192.168.122.1
```

Comment:
Made a mistake in the previous PR. This was the reason because of which I was facing internet connectivity issues. 😅